docs/components/concepts/clusters.md

```diff
@@ -0,0 +1,44 @@
+---
+id: clusters
+title: "Clusters"
+description: "Learn more about the clusters available in your Camunda Platform 8 plan."
+---
+
+A cluster is a provided group of nodes that run Camunda Platform 8. By default, Camunda Platform 8 clusters are production-ready.
+
+Professional and Enterprise plan customers can create as many production or development clusters as they want.
+
+The following table shows each plan and available type or size of cluster:
+
+|              | Development | Production - S | Production - M | Production - L |
+| ------------ | ----------- | -------------- | -------------- | -------------- |
+| Free         | \-          | \-             | \-             | \-             |
+| Free Trial   | \-          | X              | \-             | \-             |
+| Professional | X           | X              | \-             | \-             |
+| Enterprise   | X           | X              | X              | X              |
+
+## Production clusters
+
+Production clusters come in three sizes: small (S), medium (M), and large (L). To learn more about the size of cluster best suited for your use case, see our [Best Practices](/components/best-practices/best-practices-overview.md) for more information on [sizing your runtime environment](/components/best-practices/architecture/sizing-your-environment.md#sizing-your-runtime-environment).
+
+When you deploy and execute your [BPMN](/components/modeler/bpmn/bpmn.md) or [DMN](/components/modeler/dmn/dmn.md) models on a production cluster, this might impact your monthly (Professional) or annual (Enterprise) total fee, meaning the more you execute your models, the higher your total fee may be.
+
+### Free Trial cluster
+
+A Free Trial cluster is a special kind of production cluster. Free Trial clusters are size small, and only available during your trial period.
+
+Once you sign up for a Free Trial, you are able to spin up one production cluster for the time of your trial.
+
+When your trial expires, you automatically move to the Free Plan. This plan allows you to model BPMN and DMN collaboratively, but does not support execution of your models. Any cluster spun up during your free trial is deleted, and you cannot spin up new clusters.
```

Comment:
"When your trial expires, you automatically move to the Free Plan." -- I'm trying to think of a way to clarify this. For example, "When your Free Trial plan expires, you are automatically transferred to the Free Plan."

Reply:
I like this because it emphasizes the plan moving.